### ISSUE TITLE
Eevee CrowdRender fails.

### ISSUE THREAD

**Issue by deltamelter** [crowdrender/cr-docker#22]
Trying out cr-docker in in portainer on a dell R730 with Dual Xeonand 512GRAM.
First Cycles test was 5 times faster, but still not as fast as local eevee, but when running crowdrender with eevee it fails with very vague message:
Render Failed:  {'error_message': [], 'node_uuid': 'f5a413a6-fcf5-4965-xxxxxxxxxxxxx', 'node_name': 'Dock-CR', 'state': 4}

don't have a GPU... can eevee work with CPU only?

**Comment by Jeducious** [comment]
For posterity's sake, and apologies for not answering this when it was posted, the answer is, it depends.... I think.

I've run into eevee not working on other platforms, eevee requires opengl. So you need to make sure that you have it installed. On linux machines, without a display, this has been a notorious issue. Generally a headless server doesn't require any graphics libs to be installed and so opengl won't be present. That sucks cause eevee relies on opengl to do its thing.

So bottom line is, if you can install openGL, you might be able to get this to work. I know of docker containers that have opengl installed, one of those is from nvidia - [nvidia/opengl](https://hub.docker.com/r/nvidia/opengl). 

You might be able to get a container that works for eevee by starting this as a base. I'm not sure if you'd still have cuda support though since the docs for that docker image don't specify that they also support cuda. 

Hope this helps
